BitwardenShared/Core/Auth/Services/AuthService.swift

```diff
@@ -612,6 +609,48 @@ class DefaultAuthService: AuthService { // swiftlint:disable:this type_body_leng
         }
     }
 
+    /// Check if master password complies with organization policies
+    ///
+    /// - Parameters:
+    ///  - masterPassword: The user's master password
+    ///  - isPreAuth: Whether this flow is before or after authentication
+    ///  - username: The user's email address.
+    ///  - masterPasswordPolicy: The master password policies that the org has active.
+    ///
+    private func checkMasterPasswordPolicies(
+        _ masterPassword: String,
+        _ isPreAuth: Bool,
+        _ username: String,
+        _ masterPasswordPolicy: MasterPasswordPolicyResponseModel?
+    ) async throws {
```

Comment:
🎨 When multiple parameters are passed to a function we're rarely remove all parameter labels, so I would remove the `_` to be consistent with other places. Additionally, order parameters alphabetically to be consistent with other places as well.

Reply:
It's worth noting that on the whole the `_` convention was somewhat intended as a replacement for the old Objective-C `[object with:thing and:otherThing]` idea. Early Swift as I recall even turned this into `object(with:thing and:otherThing)`, which evolved into the `object(thing, and: otherThing)` that's more common today.